apps/engineering/content/design/components/empty.mdx

```diff
@@ -0,0 +1,31 @@
+---
+title: Empty
+---
+import { Empty } from "@unkey/ui"
+import { RenderComponentWithSnippet } from "@/app/components/render"
+import { Row } from "@/app/components/row"
+import { EmptyExample } from "./empty.example"
+
+## Overview
+
+The `<Empty />` component should be used when data is missing, replacing the normal component.
+
+## Empty.Icon
+
+The `<Empty.Icon />` can insert a styled icon inside the `<Empty />` component. See the full example for details.
+
+## Empty.Title
+
+The `<Empty.Title />` is used to display a title within the `<Empty />` component. It can be utilized to emphasize text or provide a warning. 
+
+## Empty.Description
+
+The `<Empty.Description />` explains why the `<Empty />` component is displayed instead of data.
+
+## Empty.Action
+
+The `<Empty.Action />` offers the user an action to resolve the lack of data. For instance, if there are too many filters on a table, a button can be added to remove those filters.
+
+## Full Example
+
+<EmptyExample/>
```

Comment:
This is not client side, you can just put the full example code here, rather than importing it from another file